spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/config/SCLayerConfig.java

```diff
@@ -0,0 +1,172 @@
+package com.boundlessgeo.spatialconnect.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class SCLayerConfig {
+
+    /**
+     * Unique id of the form.
+     */
+    @JsonProperty("id")
+    private String id;
+
+    /**
+     * Immutable name of the form.
+     */
+    @JsonProperty("layer_key")
+    private String layerKey;
+
+    /**
+     * The label to display for the form.
+     */
+    @JsonProperty("layer_label")
+    private String layerLabel;
+
+    /**
+     * The version of this form.
+     */
+    @JsonProperty("version")
+    private String version;
+
+    /**
+     * List of the form fields that define this form.
+     */
+    @JsonProperty("fields")
+    private List<HashMap<String, Object>> fields;
+
+    @JsonProperty("schema")
+    private SCLayerSchema schema;
+
+    /**
+     * Unique id of the team to which this form belongs.
+     */
+    @JsonProperty("team_id")
+    private String teamId;
+
+    /**
+     * Metadata about this form config.
+     */
+    @JsonProperty("metadata")
+    private LayerMetadata metadata;
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public List<HashMap<String, Object>> getFields() {
+        return schema.getFields();
```

Comment:
This will throw a NPE if schema is null.  Can you refactor these convenience methods or delete them (since they are already accessible in the SCLayerSchema object?